### PR TITLE
Basic changes to make the source compatible with ctapipe 10

### PR DESCRIPTION
**Pull request by FrancaCassol**
This is a preliminary step before a deeper update of this package. It should permit to read the data with the last version of ctapipe

### CODE REVIEW

**Comment by maxnoe** [comment]
@FrancaCassol You might also need to switch to github actions for the CI to run again.

See the configs in ctapipe and ctapipe io lst

**Comment by FrancaCassol** [comment]
Hi @maxnoe, 
I am afraid I am not really expert of this, of which config file are you speaking?

**Comment by maxnoe** [comment]
The files in the `.github` directory that define the workflows for github actions, they are quite similar to the `travis.yml` files.

I think taking the in `ctapipe_io_lst` as a starting point will probably work best here.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          git describe --tags
+
+
+      - name: Tests
+        run: |
+          # github actions starts a new shell for each "step", so we need to 
+          # activate our env again
+          source $CONDA/etc/profile.d/conda.sh
+          conda activate ci
+          pytest --cov=ctapipe_io_lst --cov-report=xml
```
Comment (maxnoe) [inline .github/workflows/ci.yml:51]: Change to correct module name

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        run: |
+          . $CONDA/etc/profile.d/conda.sh
+          conda config --set always_yes yes --set changeps1 no
+          sed -i -e "s/- python=.*/- python=$PYTHON_VERSION/g" environment.yml
+          conda env create -n ci -f environment.yml
+          conda activate ci
+          # we install ctapipe using pip to be able to select any commit, e.g. the current master
+          pip install pytest-cov "git+https://github.com/cta-observatory/ctapipe@$CTAPIPE_VERSION"
+          pip install -e .
+          git describe --tags
```
Comment (maxnoe) [inline .github/workflows/ci.yml:42]: this line fails as there are no tags yet in this repository

**Comment by FrancaCassol** [comment]
I close this PR and open to again from the branch update_source in the upstream repository

**Pull request closed without merging**